app/models/annotation/AnnotationRestrictions.scala

```diff
@@ -68,15 +68,12 @@ object AnnotationRestrictions {
   def readonlyAnnotation() =
     new AnnotationRestrictions {
       override def allowAccess(user: Option[User]) = true
-      override def allowFinish(user: Option[User]) = true
-      override def allowDownload(user: Option[User]) = true
     }
 
   def updateableAnnotation() =
     new AnnotationRestrictions {
       override def allowAccess(user: Option[User]) = true
       override def allowUpdate(user: Option[User]) = true
       override def allowFinish(user: Option[User]) = true
-      override def allowDownload(user: Option[User]) = true
```

Comment:
Is `allowDownload` removed from all tracings / requests? If yes, please adjust the flow types accordingly.

Reply:
No, but `allowDownload` defaults to `allowAccess` by default. So we tend to omit it whenever the two are equal.